homework01_sergio/vector.h

```diff
@@ -0,0 +1,30 @@
+class Vector {
+public:
+
```

Comment:
Did you compile this with the test file I added? I dont think it builds, there is a missing constructor